packages/p2p/src/utils/chat-message.ts

```diff
@@ -0,0 +1,77 @@
+import { FileMessage, UserMessage } from '@sendbird/chat/message';
+
+type TChatMessageArgs = {
```

Comment:
```suggestion
Interface  TChatMessageArgs  {
```

Reply:
@yauheni-deriv, did we now switch to `interfaces` instead of `types`?  😅